Composer/packages/client/src/components/CellFocusZone.tsx

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import React, { useCallback } from 'react';
+import { FocusZone, FocusZoneTabbableElements, IFocusZoneProps } from 'office-ui-fabric-react/lib/FocusZone';
+import { getFocusStyle, getTheme, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+export const formCell = css`
+  outline: none;
+  white-space: pre-wrap;
+  font-size: 14px;
+  line-height: 28px;
+`;
+
+export const formCellFocus = mergeStyles(
+  getFocusStyle(getTheme(), {
+    inset: -3,
+  })
+);
+
+/**
+ * CellFocusZone component props.
+ * Props are intently omitted as they're required for component to function correctly.
+ * Please manually test the component using keyboard in places it is used in case any changes made.
+ */
+type CellFocusZoneProps = Omit<
+  React.HTMLAttributes<unknown> & IFocusZoneProps,
+  | 'allowFocusRoot'
+  | 'data-is-focusable'
+  | 'isCircularNavigation'
+  | 'className'
+  | 'handleTabKey'
+  | 'shouldFocusInnerElementWhenReceivedFocus'
+  | 'tabIndex'
+  | 'onBlur'
+  | 'onKeyDown'
+>;
+
+const CellFocusZone: React.FC<CellFocusZoneProps> = (props) => {
+  const focusCurrentZoneAfterRender = useCallback((focusZoneEl: any) => {
+    const focusZoneId = focusZoneEl?.dataset?.focuszoneId;
```

Comment:
`focusZoneId` to follow the capitalization pattern

Reply:
Is this comment regarding `focusZoneEl?.dataset?.focuszoneId` part? If so, we can't affect this as it's how this is spelled in Fluent:

![image](https://user-images.githubusercontent.com/2841858/156447662-218a4172-9f38-4320-8b7c-4025e053c76e.png)

 the `dataset` element property does kebab case to camel case translation for us, so `data-focuszone-id` used by Fluent becomes `el.dataset.focuszoneId`,

Reply:
Ah, didn't realize this was controlled by Fluent. No problem then.